server/protocols/jmap/src/main/java/org/apache/james/jmap/model/SetFilterRequest.java

```diff
@@ -42,11 +42,17 @@ private Builder() {
         }
 
         public Builder accountId(String accountId) {
-            throw new JmapFieldNotSupportedException(ISSUER, "accountId");
+            if (accountId != null) {
```

Comment:
If I remember well, we are supposed to use `Optional.orElseThrow` in this case.

Reply:
I don't see the interest while you don't need to get/instantiate the value

Reply:
It's more

```
Optionnal.ofNullable(X).ifPresent(any -> throw new JmapFieldNotSupportedException(ISSUER, "accountId"));
```

But @rouazana  version is fine to me